File: Algorithmic-toolbox/week-1/max_pairwise_product.py
def MaxPairWiseProduct(numbers):
    results = 0
    n = len(numbers)
    for i in range(0,n):
        for j in range(i+1,n):
            if numbers[i] * numbers[j] > results:
                results = numbers[i] * numbers[j]

    return results




numList = []
num = int(input("Enter length of list: "))
for i in range(0,num):
    numList.append((int)(input()))

print(MaxPairWiseProduct(numList))

File: Algorithmic-toolbox/week-1/MaxPairWiseProduct.js
// JavaScript code
function MaxPairWiseProduct(numbers) {
    let results = 0;
    const n = numbers.length;
    for (let i = 0; i < n; i++) {
        for (let j = i+1; j < n; j++) {
            if (numbers[i] * numbers[j] > results) {
                results = numbers[i] * numbers[j];
            }
        }
    }
    return results;
}

const numList = [];
const num = parseInt(prompt("Enter length of list: "));
for (let i = 0; i < num; i++) {
    numList.push(parseInt(prompt()));
}

console.log(MaxPairWiseProduct(numList));
